src/createApi.ts

```diff
@@ -97,9 +97,11 @@ export default function createApi(
 
   return {
     getItemById,
-    getItemElementById,
+    getItemElementById: (id: string | number) =>
```

Comment:
Why do like this? Just don't understand yet.

Reply:
Because if you have two elements with the same key (used as Id in this case) the first one is always selected, also I guess is more performant just find the element inside the container and not all the document.

Reply:
Ok, possibly can be useful if have 2 ScrollMenu with same items.